authenticators/components/org.wso2.carbon.identity.sample.oauth2.federated.authenticator/pom.xml

```diff
@@ -0,0 +1,121 @@
+        </dependency>
+        <dependency>
+            <groupId>net.minidev</groupId>
+            <artifactId>json-smart</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>commons-codec.wso2</groupId>
+            <artifactId>commons-codec</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.outbound.auth.oauth2</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authenticator.oauth2</artifactId>
+            <version>${oauth2.outbound.version}</version>
+        </dependency>
+    </dependencies>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.felix</groupId>
+                    <artifactId>maven-bundle-plugin</artifactId>
+                    <version>3.2.0</version>
+                    <extensions>true</extensions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.3.1</version>
+                <inherited>true</inherited>
+                <configuration>
+                    <encoding>UTF-8</encoding>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            org.wso2.carbon.identity.sample.oauth2.federated.authenticator.internal
+                        </Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.sample.oauth2.federated.authenticator.internal,
+                            org.wso2.carbon.identity.sample.oauth2.federated.authenticator.*; version=${project.version}
+                        </Export-Package>
+                        <Import-Package>
+                            com.nimbusds.jose.util;version="${nimbusds.osgi.version.range}",
+                            net.minidev.json;version="${json-smart.version.range}"
+                            org.wso2.carbon.identity.application.authentication.framework.*;version="${carbon.identity.framework.version.range}",
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
remove dynamic import

Reply:
Addressed in https://github.com/wso2/samples-is/pull/338/commits/77e25c91e9c6a5bb5eca3e6b2743eed10d85ace1